provider/oci/common/errors.go

```diff
@@ -0,0 +1,64 @@
+// Copyright 2018 Canonical Ltd.
+// Licensed under the AGPLv3, see LICENCE file for details
+
+package common
+
+import (
+	"github.com/juju/collections/set"
+	ociCommon "github.com/oracle/oci-go-sdk/common"
+
+	envContext "github.com/juju/juju/environs/context"
```

Comment:
Only need to alias the import if there's a name clash

Reply:
Looks like I made the wrong call here. I went for consistency through the package.